### ISSUE TITLE
Checkboxes missing labels

### ISSUE THREAD

**Issue by maya** [18F/tock#521]
On the "submit your timesheet" page, the checkboxes to delete projects from your weekly list are missing labels. They'll need to be added in per https://www.w3.org/TR/WCAG20-TECHS/F68 which states that we should associate form labels with controls. 

cc: @annalee - I'm not 100% sure how to do this in Django, so could use your help.

**Comment by maya** [comment]
Closed via #523.
